parm/post_avblflds.xml

```diff
@@ -3883,7 +3883,7 @@
     <param>
        <post_avblfldidx>445</post_avblfldidx>
        <shortname>MAPS_PRMSL_ON_MEAN_SEA_LVL</shortname>
-       <pname>PRMSL</pname>
+       <pname>MSLMA</pname>
```

Comment:
@EricJames-NOAA MSLMA is a ncep local use grib2 name. You may add a line "<table_info>NCEP</table_info>" after line 3886.